.github/workflows/ci.yml

```diff
@@ -28,37 +28,37 @@ jobs:
 
     strategy:
       matrix:
-        eslint: [6.x]
-        node: [14.x]
+        eslint: [7]
+        node: [14]
         os: [ubuntu-latest]
         include:
           # On other platforms
-          - eslint: 6.x
-            node: 14.x
+          - eslint: 7
+            node: 14
             os: windows-latest
-          - eslint: 6.x
-            node: 14.x
+          - eslint: 7
+            node: 14
             os: macos-latest
           # On old Node.js versions
-          - eslint: 6.x
-            node: 12.x
+          - eslint: 7
+            node: 12
             os: ubuntu-latest
-          - eslint: 6.x
-            node: 10.x
-            os: ubuntu-latest
-          - eslint: 6.x
-            node: 8.x
+          - eslint: 7
+            node: 10
```

Comment:
Don't delete Node 8.x job.

Reply:
ESLint requires Node@^10.12.x || >12.x

Reply:
ESLint 6 works on Node.js 8.

Reply:
Ah, sorry. We have job for the minimum versions, so this is OK.